frontend/src/store/playerElement.ts

```diff
@@ -94,26 +95,35 @@ class PlayerElementStore {
   };
 
   private _setSsaTrack = async (trackSrc: string): Promise<void> => {
-    if (!subtitlesOctopus) {
-      subtitlesOctopus = new SubtitlesOctopus({
+    if (
+      !jassub &&
+      mediaElementRef.value &&
+      mediaElementRef.value instanceof HTMLVideoElement
+    ) {
+      jassub = new JASSUB({
         video: mediaElementRef.value,
         subUrl: trackSrc,
-        workerUrl: subtitlesOctopusWorkerUrl,
-        prescaleFactor: 0.5,
-        renderAhead: 90
+        workerUrl: jassubWorker,
+        legacyWorkerUrl: jassubLegacyWorker,
+        availableFonts: { 'liberation sans': jassubDefaultFont },
+        libassMemoryLimit: 40,
+        libassGlyphLimit: 40,
+        prescaleFactor: 0.8,
+        maxRenderHeight: 2160,
+        onDemandRender: false
```

Comment:
I need to add a comment to say this value is needed to work on Safari iOS/iPadOS without drawbacks on other browsers

Reply:
@ThibaultNocchi mind elaborating? to my knowledge onDemandRender has no issues on any browser outside of specifically webOS2 which lies about API support